components/identity.oauth.uma.permission.service/src/main/java/org/wso2/carbon/identity/oauth/uma/service/UMAConstants.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.uma.service;
+
+/**
+ * This class holds the constants used by Permission Endpoint.
+ */
+public class UMAConstants {
+
+    public static final String UMA_PERMISSION_ENDPOINT_CONFIG_PATH = "ptDetails.properties";
+
+    /**
+     * error descriptions
+     */
+    public enum ErrorMessages {
+        ERROR_BAD_REQUEST_INVALID_RESOURCE_ID("6001", "Permission request failed with bad resource ID."),
+        ERROR_BAD_REQUEST_INVALID_RESOURCE_SCOPE("6002", "Permission request failed with bad resource scope."),
+        ERROR_INTERNAL_SERVER_ERROR_FAILED_TO_PERSIST_PT("6003", "Error occurred while persisting PT."),
+        ERROR_INTERNAL_SERVER_ERROR_FAILED_TO_PERSIST_REQUESTED_PERMISSIONS("6004", "Error occurred while persisting" +
+                " requested permissions."),
+        ERROR_UNEXPECTED("6005", "Unexpected error.");
+
+        private final String code;
+        private final String message;
+
+        ErrorMessages(String code, String message) {
```

Comment:
Bit confused about this , why do you need to set these ?. Are these global ? This is a constant class